File: fizzbuzz.py
#!/usr/bin/env python

print('\n'.join(['FizzBuzz'[4 if i % 3 else 0:4 if i %
                            5 else 8] or str(i) for i in range(1, 51)]))

'''
def fizzbuzz(till):
    for i in range(1, till):
        if i % 15 == 0:
            print('FizzBuzz')
        elif i % 3 == 0:
            print('Fizz')
        elif i % 5 == 0:
            print('Buzz')
        else:
            print(i)
fizzbuzz(51)
'''

File: fizzbuzz.js
function fizzbuzz(till) {
    for (var i = 1; i <= till; i++) {
        if (i % 15 == 0) {
            print('FizzBuzz');
        } else if (i % 3 == 0) {
            print('Fizz');
        } else if (i % 5 == 0) {
            print('Buzz');
        } else {
            print(i);
        }
    }
}
fizzbuzz(50);
print('------------------------------------');
function fizzbuzz2(till) {
    for (var i = 1; i <= till; i++) {
        print(i % 3 == 0 ? (i % 5 == 0 ? 'FizzBuzz' : 'Fizz') : (i % 5 == 0 ? 'Buzz' : i));
    }
}
fizzbuzz2(30);